README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">
 
```

Comment:
See following link for details.

https://github.com/SixLabors/ImageSharp/blob/4e3dce1e95a57cd2a0636bbbd2784717ad994556/README.md